Makefile

```diff
@@ -475,6 +475,7 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:webhook:dir=$(WEBHOOK_ROOT) \
 		webhook
 	$(CONTROLLER_GEN) \
+		paths=./ \
```

Comment:
why the added path here?

Reply:
main.go now includes some kubebuilder annotations for RBAC related to the secure metrics endpoint changes in https://github.com/kubernetes-sigs/cluster-api/pull/9264 so this probably should have been added before in #4248.

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/9e6361a2cb433697ffd62ec5d4735e9d1309555d/main.go#L246-L248